components/nexus-core/src/test/java/org/sonatype/nexus/internal/blobstore/BlobStoreConfigurationStoreImplTest.groovy

```diff
@@ -79,7 +83,8 @@ class BlobStoreConfigurationStoreImplTest
       fail()
     }
     catch (ORecordDuplicatedException e) {
-      assert e.toString().contains('blobstore_name_idx')
+      // FIXME: This is fragile for refactoring
```

Comment:
should be sufficient now to just remove the index name check, as there's only one index that could be affected